src/lib/icon.js

```diff
@@ -9,9 +9,20 @@ const mimeTypes = {
   svg: 'image/svg+xml'
 }
 
+/**
```

Comment:
This could be replaced in the future by StackClient.getIconURL with { priority: 'registry' }

Reply:
yep. Will see that later. I don't want to mix two different refactoring task

Reply:
in #469